consensus/bor/bor.go

```diff
@@ -15,13 +16,16 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/ethereum/go-ethereum/core/rawdb"
 	lru "github.com/hashicorp/golang-lru"
 	"github.com/holiman/uint256"
 	"golang.org/x/crypto/sha3"
 
+	stakeTypes "github.com/0xPolygon/heimdall-v2/x/stake/types"
```

Comment:
Maybe we can move all the 0xPolygon imports to the end

Reply:
Done.